gradle-plugin/build.gradle

```diff
@@ -12,7 +12,7 @@ group = "com.lightbend.akka.grpc"
 def tag = "git describe --tags".execute().text.substring(1).split("-g")[0].replace("\n", "")
 def finalVersion = (tag == versionDetails().lastTag.substring(1)) ? tag : tag.reverse().reverse() + "-" + versionDetails().gitHash.substring(0, 8)
 
-version = finalVersion
+version = "1.0.2-6-a54379e8-20201005-1902" //versionDetails()//finalVersion
```

Comment:
Looks like this was accidentally committed, could you revert?

Reply:
sure, thanks.